config.yaml

```diff
@@ -1,12 +1,23 @@
+project:
+    project_name: mock_project
+    description: mock_description
+    version: mock_version
+
 input_data:
     data_location: mock
+    local_directory: mockdata/mockdata
     data_format: csv
 
-project:
-    project_name: mock_project
+configurations:
+    df_type: 'pandas'
 
 features:
-    - 'mock'
+    - location: 'https://github.com/RADAR-base-Analytics/mockfeatures'
```

Comment:
can we also add the branch name along with GitHub repo? So we can pull a particular branch the researcher want. The default branch will be `main`.

Reply:
Yeah sure. This change is also mentioned in #31.

I was thinking to add this feature with other enhancements in the next PR as this one was getting too big to track all the changes together.

Reply:
Sounds good. We can do it in the next PR.